cli-scripts/run-hadolint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+cd $(cd $(dirname $0); pwd)/../ # cd to project root
+
+if type hadolint > /dev/null 2>&1; then
+    HADOLINT="hadolint"
+elif type docker > /dev/null 2>&1; then
+    HADOLINT="docker run --rm -i hadolint/hadolint <"
+fi
+
+if [ -n "${HADOLINT}" ]; then
+    files=$(git ls-files | grep "Dockerfile")
```

Comment:
`Dockerfile`または`Dockerfile.hoge`にヒットするようにする

```suggestion
    files=$(git ls-files | grep "^Dockerfile\(\..*\)\?")
```

Reply:
ElasticSearch+日本語プラグインのためにDockerfile作るのではなく、docker runでプラグインをインストールして、インストールディレクトリを永続化することにします。
これにより、複数Dockerfileへの対応が緊急ではなくなったため、このプルリクはクローズします。